File: 1.Layers/1.3Application/1.3.2MainModule/Application.MainModule/DTOs/Mobile/PuntoVenta/PuntoVentaAsignadoDTO.cs
/*
 *PuntoVentaAsignadoDTO
 * Dto para la extración de los datos de la session para
 * el punto de venta asignado 
 * @developer Jorge Omar Tovar Martínez jorge.tovar@neoteck.com.mx
 * @date 10/12/2018 
 * @update 10/12/2018
 * @company Neoteck
 */

namespace Application.MainModule.DTOs.Mobile.PuntoVenta
{
    public class PuntoVentaAsignadoDTO
    {
        /// <summary>
        /// Id de la pipa , camioneta o estacion de carburacion 
        /// </summary>
        public short IdEstacion { get; set; }
        /// <summary>
        /// Id de su respectivo IdCAlmacenGas
        /// </summary>
        public short IdCAlmacen { get; set; }
        /// <summary>
        /// Id del punto de venta 
        /// </summary>
        public short IdPuntoVenta { get; set; }
        /// <summary>
        /// Nombre del punto de venta asignado Ej. Pipa No. 1, Isla, Camioneta No. 1
        /// </summary>
        public string  NombrePuntoVenta { get; set; }
        /// <summary>
        /// Nombre del operador asignado 
        /// </summary>
        public string NombreOperador { get; set; }
        /// <summary>
        /// Id de Operador o Chofer del usuario
        /// </summary>
        public short IdOperadorChofer { get; set; }
        /// <summary>
        /// Id de usuario 
        /// </summary>
        public short IdUusario { get; set; }
    }
}

File: 1.Layers/1.1Presentation/1.1.2Mobile/app/src/main/java/com/example/neotecknewts/sagasapp/Model/PuntoVentaAsignadoDTO.java
/*
 *  PuntoVentaAsignadoDTO
 *  Modelo DTO que permite extraer desde el api
 *  los datos del punto de venta para la generación del reporte
 *  @author Jorge Omar Tovar Martínez jorge.tovar@neoteck.com.mx
 *  @date 10/12/2018
 *  @update 10/12/2018
 *  @company Neoteck
 */
package com.example.neotecknewts.sagasapp.Model;

import com.google.gson.annotations.SerializedName;

import java.io.Serializable;

public class PuntoVentaAsignadoDTO extends RespuestaDTO implements Serializable {

    @SerializedName("IdEstacion")
    private int IdEstacion;

    @SerializedName("IdCAlmacenGas")
    private int IdCAlmacenGas;

    @SerializedName("IdPuntoVenta")
    private int IdPuntoVenta;

    @SerializedName("NombrePuntoVenta")
    private String NombrePuntoVenta;

    @SerializedName("NombreOperador")
    private String NombreOperador;

    @SerializedName("IdOperadorChofer")
    private int IdOperadorChofer;

    @SerializedName("IdUsuario")
    private int IdUsuario;

    public int getIdEstacion() {
        return IdEstacion;
    }

    public void setIdEstacion(int idEstacion) {
        IdEstacion = idEstacion;
    }

    public int getIdCAlmacenGas() {
        return IdCAlmacenGas;
    }

    public void setIdCAlmacenGas(int idCAlmacenGas) {
        IdCAlmacenGas = idCAlmacenGas;
    }

    public int getIdPuntoVenta() {
        return IdPuntoVenta;
    }

    public void setIdPuntoVenta(int idPuntoVenta) {
        IdPuntoVenta = idPuntoVenta;
    }

    public String getNombrePuntoVenta() {
        return NombrePuntoVenta;
    }

    public void setNombrePuntoVenta(String nombrePuntoVenta) {
        NombrePuntoVenta = nombrePuntoVenta;
    }

    public String getNombreOperador() {
        return NombreOperador;
    }

    public void setNombreOperador(String nombreOperador) {
        NombreOperador = nombreOperador;
    }

    public int getIdOperadorChofer() {
        return IdOperadorChofer;
    }

    public void setIdOperadorChofer(int idOperadorChofer) {
        IdOperadorChofer = idOperadorChofer;
    }

    public int getIdUsuario() {
        return IdUsuario;
    }

    public void setIdUsuario(int idUsuario) {
        IdUsuario = idUsuario;
    }
}
